### PR TITLE
Remnant Bounty Expansion

### PR DESCRIPTION
**Pull request by arkhne**
After discussion with @Zitchas on the oft-requested Dredger Bounties, I made two of them, available at later points in the current Cognizance questline.

The existing Bounties are as follows:

300k for a Raider (Crippled), unlocked after the first solo fight with a Dredger (Crippled).
400k for a Raider (Hyperdrive), unlocked after the Tech Retrieval missions.

The new Bounties are:

500k for a Dredger (Crippled), unlocked after first Fleet Hunting mission when the Korath attacked the Void Sprites.
600k for a Dredger (Hyperdrive), unlocked after the Postverta/Node events.

In order to prevent this turning into a freebie JD farm, a new variant of the Dredger was created. This variant has a Jump Drive (Broken), just like the non-crippled Raider target, in order to feed into the Broken Jump Drive Chain.

I don't have time to start up a new pilot to create a test save for this tonight, however, if this is needed, I can do this tomorrow.

### CODE REVIEW

**Comment by arkhne** [comment]
Wow, umm, yeah, this looks like it's doing what it should, but, uh, that commit history looks pretty ugly. Feel free to close this and I can see about tidying it up.

(GitHub noobness definitely showing)

**Comment by Zitchas** [comment]
I haven't fully reviewed this; Just posting to say that I welcome this additional pair of bounties and look forward to reviewing it when I'm capable of doing so.

File: data/korath/korath ships.txt [hunk 1]
```diff
@@ -110,6 +110,38 @@ ship "Korath Dredger" "Korath Dredger (Crippled)"
 	description "After the Korath were exiled from their homelands, efficient resource collection became a priority. World-ships were too precious to risk on dangerous forays, and Raiders lacked the cargo capacity. The Dredger was born as an answer to this problem, featuring an abundance of cargo and passenger space, with formidable defensive armament. With such a complement, these bulky ships can also offer potent support for frontline Raiders."
 
 
+ship "Korath Dredger" "Korath Dredger (Hyperdrive)"
+	sprite "ship/dredger"
```
Comment (ravenshining) [inline data/korath/korath ships.txt:114]: You aren't changing the sprite so this isn't necessary.
```suggestion
```

File: data/korath/korath ships.txt [hunk 2]
```diff
@@ -110,6 +110,38 @@ ship "Korath Dredger" "Korath Dredger (Crippled)"
...
+	turret 37.5 -152.5 "Korath Warder"
+	turret 0 -153 "Korath Grab-Strike"
+	turret -42 -28 "Korath Banisher"
+	turret 42 -28 "Korath Banisher"
+	turret -86 128.5 "Korath Grab-Strike"
+	turret 86 128.5 "Korath Grab-Strike"
+	engine 0 179
+	engine -32 164 .7
+	engine 32 164 .7
+	description "After the Korath were exiled from their homelands, efficient resource collection became a priority. World-ships were too precious to risk on dangerous forays, and Raiders lacked the cargo capacity. The Dredger was born as an answer to this problem, featuring an abundance of cargo and passenger space, with formidable defensive armament. With such a complement, these bulky ships can also offer potent support for frontline Raiders."
```
Comment (ravenshining) [inline data/korath/korath ships.txt:143]: Not necessary to specify all this which will be carried over from the regular dredger.
```suggestion
```

File: data/remnant/remnant missions.txt [hunk 3]
```diff
@@ -3674,6 +3674,33 @@ mission "Remnant: Cognizance 4"
...
+mission "Remnant: Bounty 3"
+	job
+	repeat
+	name "Remnant bounty"
+	description "Hunt down a crippled Korath ship that is lurking in Remnant territory, then return to <planet> to receive your payment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Cognizance 4: failed"
+		random < 60
```
Comment (ravenshining) [inline data/remnant/remnant missions.txt:3687]: Dredger bounties ought to be rarer than Raider bounties.

```suggestion
		random < 20
```

File: data/remnant/remnant missions.txt [hunk 4]
```diff
@@ -4024,6 +4051,33 @@ mission "Remnant: Cognizance 14"
...
+mission "Remnant: Bounty 4"
+	job
+	repeat
+	name "Remnant bounty"
+	description "Hunt down a Korath ship that is lurking in Remnant territory, then return to <planet> to receive your payment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Cognizance 14: done"
+		random < 60
```
Comment (ravenshining) [inline data/remnant/remnant missions.txt:4063]: Dredger bounties ought to be rarer than Raider bounties.

```suggestion
		random < 20
```

**Review by ravenshining — CHANGES_REQUESTED** [review]

**Comment by ravenshining** [comment]
If you accept the above changes, perhaps remember to PR them to Revamp as https://github.com/Zitchas/endless-sky/pull/93 is already merged there.

File: data/remnant/remnant missions.txt (hunk 4 above)
Comment (arkhne) [reply data/remnant/remnant missions.txt:4063]: This is a good point. However, I think 20 is too low. Should we try for 30 or 40?

**Comment by arkhne** [comment]
So, I've been in active discussion with a few people trying to figure out the best way to proceed here. I've also got a few more changes (decreasing the payment and frequency) in the works, and slight tweaks to the naming scheme and descriptions to make it more flavoursome now that it's being expanded.

Just wanted to give an update in case anybody is wondering what's taking so long. Remnant is Zitchas' domain, so I make certain not to step on his toes and discuss changes with him first, and make sure they are both acceptable, and don't conflict with future plans (or sometimes even do some of that future work, if I can). This is not dead, I'm just making sure it's done right.

**Comment by arkhne** [comment]
I am marking the conversations resolved, however, I will be in continued contact with @ravenshining and others in attempts to make this even better.

**Review by Zitchas — APPROVED** [review]
I haven't really dug into the code yet, but the concept and initial read look good. I like this expansion of the Remnant bounties.

(Just for the record, Arkhne talked these over with me before they PR'd them, so this wasn't really a surprise)

**Review by Amazinite — COMMENTED** [review]
The PR description says that the new job payments are 500k and 600k, yet I see 400k and 500k. Is this intentional?

**Comment by arkhne** [comment]
> The PR description says that the new job payments are 500k and 600k, yet I see 400k and 500k. Is this intentional?

Yes. The payments were reducing when the in-game descriptions were added to all the bounties, as the high payout was proving a source of disagreement with some. Originally, the payouts were because all the bounties basically had the same descriptions, so they needed to have noticable payout differences (which meant higher). Once some editing of descriptions was done to improve their flavour and give them an individual identity, that was less needed, and I was able to address payout concerns.

**Pull request merged**